pytorch_to_returnn/torch/_C.py

```diff
@@ -122,17 +124,57 @@ class Size(tuple):  # type: Tuple[SizeValue, ...]
 
 class SizeValue(int):
   """
-  We extend this, to store extra information, e.g. such that this reflects the batch-dim.
+  We extend this, to store extra information, e.g. corresponding RETURNN dim tags.
   """
-  is_batch_dim: bool = False
-  merged_dims: list[SizeValue] = []
+  def __new__(cls, x: int, dim_tag: Optional[Dim] = None, merged_dims: Optional[List[SizeValue]] = None,
+              originating_tensor=None):
+    res = super(SizeValue, cls).__new__(cls, x)
+    res.dim_tag = dim_tag or Dim(dimension=x, description="static_dim")
+    res.merged_dims = merged_dims or []
+    res.originating_tensor = originating_tensor
+    return res
+
+  @property
+  def is_batch_dim(self):
+    if self.dim_tag is None:
+      return False
+    return self.dim_tag.is_batch_dim()
+
+  @property
+  def axis(self):
+    naming = Naming.get_instance()
+    return naming.tensors[self.originating_tensor].returnn_data.get_axis_by_tag_name(self.dim_tag.description)
```

Comment:
Do not use `get_axis_by_tag_name`, and also never use `Dim.description`.
Instead, use `get_axis_from_description` (the method name is maybe confusing with `Dim.description` but this is not meant here; this accepts a generic axis description, or directly a dim tag).
Also, you return the RETURNN axis here, which is wrong! You should return the PyTorch axis.
So:
```suggestion
    tensor_entry = naming.tensors[self.originating_tensor]
    returnn_axis = tensor_entry.returnn_data.get_axis_from_description(self.dim_tag)
    return tensor_entry.torch_axis_from_returnn_axis[returnn_axis]
```